Sources/eppo/EppoClient.swift

```diff
@@ -115,6 +125,10 @@ public class EppoClient {
             assignmentCache: assignmentCache,
             initialConfiguration: initialConfiguration
         )
+        
+        if let callback = configurationChangeCallback {
```

Comment:
why not pass the callback to initializeOffline since it notifies at the ends of its routine as well?

Reply:
Yep, it's there too

https://github.com/Eppo-exp/eppo-ios-sdk/blob/2e4fe3c3da0b24e08212ddf561294526dbf31820/Sources/eppo/EppoClient.swift#L78-L102

Reply:
Good idea! That would be less repetitive